### PR TITLE
Fix Prometheus data source secret generation

### PR DESCRIPTION
**Pull request by anishasthana**
Signed-off-by: Anish Asthana <anishasthana1@gmail.com>

### CODE REVIEW

**Review by crobby — APPROVED** [review]
works for me, the secret seems to be created correctly

File: monitoring/grafana-dashboards/jupyterhub-slis.configmap.yaml [hunk 1]
```diff
@@ -371,15 +371,15 @@
                   {
                     "expr": "sum(jupyterhub_server_spawn_duration_seconds_count{status=\"success\"})/sum(jupyterhub_server_spawn_duration_seconds_count{})",
                     "interval": "",
-                    "legendFormat": "",
+                    "legendFormat": "Jupyterhuv Server Spawn Success Rate",
```
Comment (crobby) [inline monitoring/grafana-dashboards/jupyterhub-slis.configmap.yaml:374]: Jupyterhuv??? :)

**Review by crobby — CHANGES_REQUESTED** [review]
Probably just the one jupyterhuv type-o

File: monitoring/grafana-dashboards/jupyterhub-slis.configmap.yaml (hunk 1 above)
Comment (anishasthana) [reply monitoring/grafana-dashboards/jupyterhub-slis.configmap.yaml:374]: arghhh

Comment (anishasthana) [reply monitoring/grafana-dashboards/jupyterhub-slis.configmap.yaml]: fixed

**Review by crobby — APPROVED** [review]
appears to be working for me; lgtm

**Pull request merged**